content/password.md

```diff
@@ -84,4 +86,52 @@ exit
 reboot
 ```
 
-Now, enter the original passphrase. Here you will be prompted for the new passphrase, and then to confirm the new passphrase.
+## Changing LUKS passphrase
+
+If you would like to change the password on a LUKS encrypted partition, run this command (replace the example partition with your root partition):
+
+```bash
+cryptsetup luksChangeKey /dev/sda3 -S 0
+```
+
+Enter the original pass-phrase, and follow the prompts for the new pass-phrase ( and to confirm the new pass-phrase).
+
+## Adding LUKS passphrases
+
+Full Disk Encryption supports up to 8 total pass-phrases to unlock the encryption. This is useful if you would like to allow more than a single user to unlock the encryption and not share a pass-phrase. The default pass-phrase is in slot 0, with space for additional pass-phrases in slots 1 through 7 (for a total of 8).
+
+### List Partitions
+
+First let's list the partitions of all of the installed drives:
+
+```
+lsblk -f
```

Comment:
While it is more to type, I would be inclined to replace `lsblk -f` with `lsblk -o name,mountpoints` as it strips out a large quantity of unnecessary information, and will not cause incredibly hard-to-parse line-wrapping to occur even on smaller terminal windows (like the default terminal window size when spawned as a floating window)